client/src/main/java/org/spine3/base/Commands.java

```diff
@@ -53,7 +52,7 @@
     /**
      * A substring which the {@code .proto} file containing commands must have in its name.
      */
-    public static final String COMMANDS_FILE_SUBSTRING = "commands";
+    public static final String COMMANDS = "commands";
```

Comment:
“commands” what? From the API point of view it looks this `Commands.COMMANDS`?

Why don't we have it as `Commands.FILE_SUFFIX`, for example?